notebooks/en/labelling_feedback_setfit.ipynb

```diff
@@ -0,0 +1,2842 @@
+{
```

Comment:
<ul><li>For the title, it might be good to mirror this to the title in the TOC (Add zero-shot text classification suggestions using SetFit). </li><li>I think it could also be useful to have a little bit more background context, on Argilla and what you are aiming to achieve and why someone might follow the tutorial. Something like "Argilla is an open tool for..." </li><li>(personal preference) I think it's nice to give some overview of what someone will learn from following the tutorial. </li></ul><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/56/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>